fleetspeak/src/inttesting/integrationtest/frr.go

```diff
@@ -79,7 +79,7 @@ func (c *statsCounter) MessageSaved(forClient bool, m *fspb.Message) {
 	}
 }
 
-func (c *statsCounter) MessageProcessed(start, end time.Time, service, messageType string) {
+func (c *statsCounter) MessageProcessed(start, end time.Time, service string, m *fspb.Message) {
```

Comment:
service is also extractable from message, so we should also remove it as a parameter - if the server is processing a message, then the service processing it should be m.Destination.ServiceName